vue/components/ui/organisms/form/form.vue

```diff
@@ -299,6 +309,16 @@ export const Form = {
             };
             return base;
         },
+        sectionClasses(index) {
+            return `section-${index}`;
+        },
+        formInputClasses(field) {
+            const base = {};
+            base[`field-${field.value}`] = Boolean(field.value);
+            base[`field-${field.type}`] = Boolean(field.type);
+            base[`field-${field.meta}`] = Boolean(field.meta);
+            return base;

```

Comment:
Let's go with the same old approach where possible

```suggestion
            const base = {};
            base[`field-${field.value}`] = true;
            base[`field-${field.type}`] = true;
            base[`field-${field.meta}`] = Boolean(field.meta);
            return base;

```